### PR TITLE
chore: Fix nginx rewrite path error when create deployment via BYOC

### PR DESCRIPTION
**Pull request by xieydd**

### CODE REVIEW

File: ingress-operator/pkg/controller/core.go [hunk 1]
```diff
@@ -267,11 +269,18 @@ func MakeAnnotations(fni *faasv1.InferenceIngress, host string) map[string]strin
 					"/" + fni.Spec.Function + "/$1"
 				annotations["nginx.ingress.kubernetes.io/use-regex"] = "true"
 			default:
-				annotations["nginx.ingress.kubernetes.io/rewrite-target"] = "/inference/" + fni.Name + "." + inferenceNamespace + "/$1"
-				annotations["nginx.ingress.kubernetes.io/ssl-redirect"] = "false"
-				annotations["nginx.ingress.kubernetes.io/use-regex"] = "true"
+				// for inference created by modelz apiserver
```
Comment (cutecutecat) [inline ingress-operator/pkg/controller/core.go:272]: `case "apiserver"` seems fit for inference created by modelz apiserver, could we strightly use it for revise?

Comment (cutecutecat) [inline ingress-operator/pkg/controller/core.go:272]: Is this designed for BYOC created inference when `INGRESS_OPERATOR_SERVICE_HOST` might be not be apiserver?

Comment (xieydd) [reply ingress-operator/pkg/controller/core.go:272]: If you set `INGRESS_OPERATOR_SERVICE_HOST` be `apiserver`, you can not use `mdz` create inference in local environment.

**Comment by xieydd** [comment]
`nginx.ingress.kubernetes.io/ssl-redirect` is necessary for deployment which created in modelz agent? @gaocegege

**Comment by gaocegege** [comment]
Yeah I think so.

**Comment by xieydd** [comment]
> Yeah I think so.

So I think we still need to keep `host`.

**Pull request merged**